acos_client/v30/slb/virtual_port.py

```diff
@@ -98,3 +101,12 @@ def delete(self, virtual_server_name, name, protocol, port):
         url = self.url_server_tmpl.format(name=virtual_server_name)
         url += self.url_port_tmpl.format(port_number=port, protocol=protocol)
         return self._delete(url)
+
+    def _set_sampling_enable(self, sample_list, dest_obj):
+        dest_array = []
+        if len(sample_list) > 0:
```

Comment:
How can sampling-enable be set to nothing (`[]`) if it's only submitted when it's not nothing?